hq_superset/hq_domain.py

```diff
@@ -44,6 +44,7 @@ def after_request_hook(response):
     'AuthOAuthView.login',
     'AuthOAuthView.logout',
     'AuthOAuthView.oauth_authorized',
+    'CurrentUserRestApi.get_me',
```

Comment:
Curious what this is for?

Reply:
From the UI, this a call made to constantly sync the user record, I assume its getting used in javascript/UI somewhere.
It is a domain independent API call.
I added it to this list so we don't fire the before request hooks on such requests since they are not needed.

Reply:
Found one more, https://github.com/dimagi/commcare-analytics/pull/72/commits/b4eb1b3b1745278ea700f8fa5c98bf9109fa09b2